lib/linguist/languages.yml

```diff
@@ -2668,6 +2669,16 @@ OpenSCAD:
   tm_scope: none
   ace_mode: scad
 
+OpenType Font Feature:
+  type: programming
```

Comment:
PostScript? Uh, no, PostScript *is* a fully Turing-complete programming language... CSS, yes, that's markup.

I'm well aware of what OpenType features are, but I'd argue they're no closer to programming than SVG source is. They describe coordinates, geometry and hinting for the rendering environment...